aerospike-proximus/examples/gke/as-proximus-gke-values.yaml

```diff
@@ -25,9 +25,11 @@ proximusConfig:
     ticker-interval: 10
 
 service:
-  enabled: true
+  enabled: false
+  tyoe: LoadBalancer
```

Comment:
@davi17g did this work? It looks like a typo

Reply:
@arrowplum It is because LB is on istio side, it's should be like that

Reply:
"tyoe"?